i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/data-operate/import/import-way/broker-load-manual.md

```diff
@@ -44,30 +70,103 @@ BE 在执行的过程中会从 Broker 拉取数据，在对数据 transform 之
 
 当前 BE 内置了对 HDFS 和 S3 两个 Broker 的支持，所以如果从 HDFS 和 S3 中导入数据，则不需要额外启动 Broker 进程。如果有自己定制的 Broker 实现，则需要部署相应的 Broker 进程。
 
-## 导入语法
+## 快速上手
+
+本节演示了一个 S3 Load 的例子。具体的使用语法，请参考 SQL 手册中的 [Broker Load](../../../sql-manual/sql-statements/Data-Manipulation-Statements/Load/BROKER-LOAD)。
```

Comment:
../../../sql-manual/sql-statements/data-modification/load-and-export/BROKER-LOAD.md